src/reason_pprint_ast.ml

```diff
@@ -5006,7 +5320,7 @@ class printer  ()= object(self:'self)
         )
     | Pmod_structure (s) ->
       makeList
-        ~break:IfNeed
+        ~break:Always_rec
```

Comment:
@jordwalke Don't know what's happening here. But I have to force break this for the case of 
```
let module AMod = {
  let a = 10;
};
```